Gandalan.IDAS.WebApi.Client/RESTRoutinen.cs

```diff
@@ -61,18 +66,19 @@ public string UserAgent
         /// <param name="url">Relative URL, bezogen auf die BaseUrl</param>
         /// <param name="settings"></param>
         /// <returns>Objektinstanz</returns>
-        public async Task<T> GetAsync<T>(string url, JsonSerializerSettings settings = null)
+        public async Task<T> GetAsync<T>(string url, JsonSerializerSettings settings = null, string version = null)
```

Comment:
Please fix new warnings (`Parameter 'version' has no matching param tag in the XML comment`)